websites/websites.model.lkml

```diff
@@ -1,2 +1,4 @@
 connection: "telemetry"
-label: "Websites"
\ No newline at end of file
+label: "Websites"
+
+include: "/websites/views/*.view.lkml"
```

Comment:
You still need to add the explores. For [example ](https://github.com/mozilla/looker-spoke-default/blob/main/firefox_accounts/firefox_accounts.model.lkml)

Reply:
The explores are what the end users see in looker and pick from

Reply:
Added it. Now I can see these views when going into `Explore`